core/src/main/java/site/ycsb/measurements/Measurements.java

```diff
@@ -127,6 +127,7 @@ public Measurements(Properties props) {
   }
 
   private OneMeasurement constructOneMeasurement(String name) {
+    System.out.println("Constructing one measurement type " + measurementType + " for " + name);
```

Comment:
can we remove all those debuging print ?, or align them with how ycsb is doing logging/debug prints ?